protoc-gen-kotlin/protoc-gen-kotlin-common/src/main/kotlin/pbandk/gen/pb/plugin.kt

```diff
@@ -149,6 +227,30 @@ private fun CodeGeneratorRequest.Companion.protoUnmarshalImpl(protoUnmarshal: pb
     }
 }
 
+fun CodeGeneratorRequest.toJsonMapper(): CodeGeneratorRequest.JsonMapper =
+    CodeGeneratorRequest.JsonMapper(
+        fileToGenerate,
+        parameter,
```

Comment:
Yes, you are probably right... I actually didn't get around to examining this behavior in Postman with streem-service... this would've been obvious very quickly, and will fix, assuming it is not doing the right thing.

I would've liked to do more testing/QA on this before sending this PR up, but thank you for looking at it.